pyirf/perf/irf_maker.py

```diff
@@ -523,6 +561,58 @@ def _make_hdu(cls, hdu_name, t, cols):
         hdu.header.set("EXTNAME", hdu_name)
         return hdu
 
+    @classmethod
+    def _make_aeff_hdu(cls, table_energy, table_theta, aeff):
+        """List of columns"""
+        hdu = fits.BinTableHDU.from_columns(
+            [
+                fits.Column(
+                    name="ETRUE_LO",
+                    format=table_energy["ETRUE_LO"].format,
+                    unit=table_energy["ETRUE_LO"].unit.to_string(),
+                    array=np.atleast_2d(table_energy["ETRUE_LO"]),
+                ),
+                fits.Column(
+                    "ETRUE_HI",
+                    table_energy["ETRUE_HI"].format,
+                    unit=table_energy["ETRUE_HI"].unit.to_string(),
+                    array=np.atleast_2d(table_energy["ETRUE_HI"]),
+                ),
+                fits.Column(
+                    "THETA_LO",
+                    table_theta["THETA_LO"].format,
+                    unit=table_theta["THETA_LO"].unit.to_string(),
+                    array=np.atleast_2d(table_theta["THETA_LO"]),
+                ),
+                fits.Column(
+                    "THETA_HI",
+                    table_theta["THETA_HI"].format,
+                    unit=table_theta["THETA_HI"].unit.to_string(),
+                    array=np.atleast_2d(table_theta["THETA_HI"]),
+                ),
+                fits.Column(
+                    "AEFF",
+                    aeff["AEFF"].format,
+                    unit=aeff["AEFF"].unit.to_string(),
+                    array=np.expand_dims(aeff["AEFF"].data, 0),
+                    dim=str(aeff["AEFF"].data.T.shape),
+                ),
+            ]
+        )
+
+        hdu.header.set(
+            "TDIM7",
+            "("
+            + str(len(table_energy["ETRUE_LO"]))
+            + ","
+            + str(len(table_theta["THETA_LO"]))
+            + ")",
+        )
+        hdu.header.set(
+            "EXTNAME", "AEFF", "name of this binary table extension "
```

Comment:
The recommended EXTNAME keyword is “EFFECTIVE AREA”.
from https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/full_enclosure/aeff/index.html

Reply:
The specific nomenclature depends on the version of the GADF.
We should decide which one do we want to work with.
As a preliminary thought, in issue #6 I proposed to use the latest stable release instead of the very latest development one.
In that case this name would be instead `EFFECTIVE AREA (RECO)`.

Reply:
ok but on my new commit this is already change.....

Reply:
sure no problem!
I just wanted to recall that we didn't fix this choice for now (if for some reason it is obvious and I missed it!)
In any case, this is a detail about the output format, so it is not critical for now.

Reply:
yes I agree!!!

Reply:
So we can AEFF for now?